daemon/daemon.go

```diff
@@ -760,9 +760,23 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					if c.IsPaused() {
```

Comment:
@tiborvass  @jhowardmsft  Do you mean we should add a empty `SignalMap` in `signal_windows.go`?
and we get the signal by `sig, ok := signal.SignalMap["TERM"]` ?